contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,408 @@
+    properties: {
+        timezone: 'GMT+8',
+    },
+})
+
+/* User A will now have properties:
+{ 
+  name: 'User A',
+  location: 'London', 
+  phone: '0800-POSTHOG', 
+  timezone: 'GMT+8' 
+}
+*/
+```
+
+### How to handle duplicates of the same user
+
+It may happen that, due to implementation issues, the same user in your product has multiple users in PostHog associated with them. In these cases, you can use `$merge_dangerously` to merge multiple PostHog users into a single user.
+   
+> **Warning:** Merging users with `merge_dangerously` is irreversible and has no safeguards! Be careful not to merge users who should not be merged together.
+> 
+> Due to the dangers, we don't recommend you merge users frequently, but rather as a one-off for recovering from implementation problems. We also encourage you to consider whether merging users is strictly necessary for your data integrity, or a 'nice to have'.
+
+Merging users is done by sending a `$merge_dangerously` event:
+
+<MultiLanguage selector="tabs">
+
+```node
+client.capture({
+    distinctId: 'distinct_id_of_user_to_merge_into',
+    event: '$merge_dangerously',
+    properties: {
+        alias: 'distinct_id_of_user_to_be_merged',
+    },
+})
+```
+
+```py
+posthog.capture('distinct_id_of_user_to_merge_into', 
+  '$merge_dangerously', 
+  {'alias': 'distinct_id_of_user_to_be_merged'})
+```
+
+```go
+client.Enqueue(posthog.Capture{
+  DistinctId: "distinct_id_of_user_to_merge_into",
+  Event:      "$merge_dangerously",
+  Properties: posthog.NewProperties().
+    Set("alias", "distinct_id_of_user_to_be_merged"),
+})
+```
+
+```ruby
+posthog.capture({
+    distinct_id: 'distinct_id_of_user_to_merge_into',
+    event: '$merge_dangerously',
+    properties: {
+        alias: 'distinct_id_of_user_to_be_merged'
+    }
+})
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+      "alias": "distinct_id_of_user_to_be_merged"
+    },
+    "timestamp": "2020-08-16 09:03:11.913767",
+    "distinct_id": "distinct_id_of_user_to_merge_into",
+    "event": "$merge_dangerously"
+}' https://app.posthog.com/capture/
+```
+
+</MultiLanguage>
+
+#### How are properties managed when merging users?
+
+
+When a `User B` is merged into another `User A`, all the properties of the `User B` are added to `User A`. If there is a conflict, the properties of `User A` are prioritized over `User B`. For example:
+```js
+client.identify({
+    distinctId: 'user_A',
+    properties: {
+        location: 'London',
+        name: 'User A',
+    },
+})
+
+client.identify({
+    distinctId: 'user_B',
+    properties: {
+        name: 'User B',
+        location: 'Rome',
+        phone: '0800-POSTHOG',
+    },
+})
+
+posthog.alias(
+  'user_A', // distinct_id
+  'user_B'  // alias_id
+)
+
+/* User B has merged into User A. User A will now have properties:
+{ 
+  name: 'User A',
+  location: 'London', 
+  phone: '0800-POSTHOG', 
+}
+*/
+```
```

Comment:
This doesn't work for several reasons:
1. alias_id you use is already identified user (whenever you use a distinct_id as distinct_id in identify call it is now `is_identified` even if an actual merge didn't happen)
2. calling identify in a row the second call would merge the users already (even if you said imagine different browsers it's still simpler to just call out the properties state).
3. This shows a bad use case of merging 2 users based on properties looking like they belong to different users

I suggest we call out the likely reasonable use case (rather than merging users, also move it to be after identify in the FAQ so it's more logical ordering. 
Furthermore let's not use js here as the alias call language for clarity.

```suggestion
#### How are properties managed when assigning multiple distinct IDs to the same user?

When you use alias or `$merge_dangerously` to merge 2 distinctIDs that represent the same user, then sometimes both of those PostHog users have properties. When a `distinct_id_B` is merged into another `distinct_id_A`, all the properties of the `distinct_id_B` are added to `distinct_id_A`. If there is a conflict, the properties of `distinct_id_A` are prioritized over `distinct_id_B`. For example:
```node
/* distinct_id_A user in PostHog has these properties
{
        location: 'London',
        name: 'Max the Hedgehog',
        favorite_food: 'pineapple'
},
distinct_id_B user in PostHog has these properties
{
        phone: '0800-POSTHOG',
        favorite_food: 'pizza'
}
*/
client.alias({
    distinctId: 'distinct_id_A',
    alias: 'distinct_id_B',  // if both users have been identified or aliased in the past use `$merge_dangerously`
})

/* The resulting user will now have properties:
{ 
  name: 'Max the Hedgehog',
  location: 'London', 
  phone: '0800-POSTHOG', 
  favorite_food: 'pineapple'
}
*/
```
```

Reply:
Hmm you're right, this is not a good example.  We should use `merge_dangerously` code instead of `alias` for this example.

> also move it to be after identify in the FAQ so it's more logical ordering.

Im not sure what you mean by this.

> #### How are properties managed when assigning multiple distinct IDs to the same user?

I think its simpler to separate the `alias` merge from the `$merge_dangerously` merge, since the `alias` merge applies to anonymous users only right?

Reply:
for properties merging it doesn't matter if it was anonymous user or not

Reply:
> also move it to be after identify in the FAQ so it's more logical ordering.

FAQ:
## how anonymous properties are merged
## how added distinct ID properties are merged (these are the same actually)
## other sections